app/components/CustomIntegrationDemo.tsx

```diff
@@ -0,0 +1,73 @@
+'use client';
+import { Modal, ModalBody, ModalContent, ModalHeader } from '@nextui-org/react';
+import Image from 'next/image';
+import { useState } from 'react';
+import OrderHistoryIcon from '../assets/orderHistory.svg';
+import { useCoinbaseRampTransaction } from '../contexts/CoinbaseRampTransactionContext';
+import { AmountInput } from './AmountInput';
+import { ChainTokenSelector } from './ChainTokenSelector';
+import { CurrencySelector } from './CurrencySelector';
+import { OrderHistory } from './OrderHistory';
+import { RampTransactionSummary } from './RampTransactionSummary';
+import { RegionSelector } from './RegionSelector';
+import { WalletConnector } from './WalletConnector';
+
+export const CustomIntegrationDemo = () => {
+  const [showOrderHistory, setShowOrderHistory] = useState(false);
+  const { authenticated } = useCoinbaseRampTransaction();
+
+  return (
+    <>
+      <div className="flex gap-4 justify-center py-4">
+        <WalletConnector />
+        {authenticated && (
+          <Image
+            onClick={() => setShowOrderHistory(true)}
+            className="cursor-pointer hover:opacity-50 active:opacity-30"
+            src={OrderHistoryIcon}
+            width={24}
+            height={24}
+            alt="Order History"
+            aria-label="Order History"
+          />
+        )}
+      </div>
+      <div className="flex flex-col gap-4 justify-center items-center">
+        <div>
+          <RegionSelector />
+        </div>
+        <div className="flex flex-col md:flex-row justify-around min-h-[750px] max-w-screen md:w-[1000px] m-auto gap-4 md:gap-0">
+          <div className="flex flex-col gap-4 grow">
+            <div>
+              <AmountInput />
+              <div className="flex flex-col md:flex-row gap-4 items-center justify-around">
+                <ChainTokenSelector />
+                <CurrencySelector />
+              </div>
+            </div>
```

Comment:
```suggestion
            <AmountInput />
            <div className="flex flex-col md:flex-row gap-4 items-center justify-around">
              <ChainTokenSelector />
              <CurrencySelector />
            </div>
```
Can this div be removed? @rustam-cb